libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -1385,7 +1388,11 @@ void IRGeneratorForStatements::endVisit(FunctionCall const& _functionCall)
 		t("saltSet", functionType->saltSet());
 		if (functionType->saltSet())
 			t("salt", IRVariable(_functionCall.expression()).part("salt").name());
-		t("retVars", IRVariable(_functionCall).commaSeparatedList());
+		solAssert(IRVariable(_functionCall).stackSlots().size() == 1, "");
+		t("address", IRVariable(_functionCall).commaSeparatedList());
+		t("isTryCall", _functionCall.annotation().tryCall);
+		if (_functionCall.annotation().tryCall)
+			t("success", IRNames::trySuccessConditionVariable(_functionCall));
```

Comment:
So this function creates a new variable for `_functionCall` which is set via `success`, and potentially reused later via this same variable `IRNames::trySuccessConditionVariable(_functionCall)`?

Reply:
Yes, the try-catch mechanism uses the same function to access the name of the variable.